### DISCUSSION TITLE
WSA 2407.40000.0.0

### DISCUSSION THREAD

**Discussion by Bob3500** [microsoft/WSA#583] [General]
Hi all,
I just see that my WSA has been updated to version 2407.40000.0.0. What is it ???!!???
- WSA revival with fixes and new features ???
- or just the version that will automatically disable itself in March 2025 ??

![image](https://github.com/user-attachments/assets/6d14d60c-f002-4843-8163-d8796b3b25c6)

**Comment by raaomoka** [comment]
Yeah, I'm curious to know what's new here. Wary to install anything after WSA v2310.40000.2.0, as that's been working fine on my Surface Pro 7+ with minimal issues...

**Comment by MNLierman** [comment]
I'm asking for myself, and for everyone else, [over at MustardChef/WSABuilds](https://github.com/MustardChef/WSABuilds/discussions/424). Some many say that this is a conspiracy and not take such questions seriously, but they would be ignorant to suggest that. All the time we see services and apps that are scheduled to be decommissioned, have a final build released with a few "bug fixes" and a kill switch added. I personally don't like my work to be disturbed, so I turn off automatic app updates on my phone, and quite frequently find myself downloading old versions of apps from months or weeks prior due to some issue, etc. I don't know how anyone else deals with this, or if they just adapt and if they used something very heavily and it's been axed, they just move on? My life is far too busy for that.

Over the years, when things are announced to be decommissioned, I check to make sure that updates for that is off, and I make a mental note not to update it manually. Evernote – a note taking and documentation software, they've had a free plan for 15 years, and I used that off and on half the time paying for the yearly subscription. They announced end of last year that they would be disabling the ability to add or edit notes for any accounts that have more than 50 notes or pages saved. So I rolled back to a version a few weeks prior to that announcement, and rocked that build for a couple of months until they disabled all ability to use old versions. I was able to continue using the app the way I had been on that old version because it didn't have a kill switch. So this is just one small recent example in my life, but there are so many out there.

**Reply by SapientSi** [reply]
When a program is announced to no longer be supported, it always feels quite regrettable. RIP
